recipes/DataProfiler/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set name = "DataProfiler" %}
+{% set version = "0.8.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DataProfiler-{{ version }}.tar.gz
+  sha256: f544b0ab58679ac4c93155439f6fd817241fb480290284a2dde8ce5b6b9e7846
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
```

Metadata says it is 3.8

```suggestion
    - python >=3.8
```